MagaluCloud/magalu
# Fixes on create links script

## Description
Some changes that were applyed:
- [x] Resolving $refs in action parameters
- [x] Fixing update links to wrong resources
- [x] Better link naming
- [x] Using the jsonPointer resolver to resolve referenced schemas
- [x] Using tree structure to get related paths when building links
- [x] Easier debugging with the func to print the tree to use as reference

## How to test

- Get an openapi.yaml
- Run the `scripts/transformer/transform.py` on it and check the result file
- To make it easier to analyze which paths are related, call the print_tree function in `spec_create_links.py` to see the tree structure of the paths
